robocop/checkers/naming.py

```diff
@@ -290,6 +297,12 @@
         reports any occurrence of `@[` characters.
         """,
     ),
+    "0321": Rule(
+        rule_id="0321",
+        name="missing-4-spaces-before-argument",
```

Comment:
If we decide on configurable number of spaces, name of the rule will be invalid. RF also supports tabs and pipes. It should be ``missing-whitespace-before-argument`` or similar.